presidio-structured/setup.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# noqa: D100
+import os.path
+from os import path
+
+from setuptools import setup, find_packages
+
+test_requirements = ["pytest>=3", "flake8==3.7.9"]
+
+__version__ = ""
+this_directory = path.abspath(path.dirname(__file__))
+parent_directory = os.path.abspath(os.path.join(this_directory, os.pardir))
+
+with open(path.join(this_directory, "README.MD"), encoding="utf-8") as f:
+    long_description = f.read()
+
+try:
+    with open(os.path.join(parent_directory, "VERSION")) as version_file:
+        __version__ = version_file.read().strip()
+except Exception:
+    __version__ = os.environ.get("PRESIDIO_VERSION", "0.0.1-alpha")
+
+setup(
+    name="presidio_structured",
+    python_requires=">=3.5",
+    version=__version__,
+    packages=find_packages(
```

Comment:
Can you try replacing this line with `packages=find_packages(exclude=["tests"]),`?